docs/integrations/cloudwatch/list-cloudwatch-logs.rst

```diff
@@ -0,0 +1,6 @@
+Send logs to Amazon CloudWatch
+==============================
+
+Browse through information related to sending your Aiven service logs to your `Amazon (AWS) CloudWatch <https://aws.amazon.com/cloudwatch/>`_.
```

Comment:
Should there be some files for this to link to?  It looks like an empty section at the moment.